database/migrations/2021_01_11_231447_migrate_swag_data.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+declare(strict_types=1);
+
+use App\Models\DuesTransaction;
+use App\Models\FiscalYear;
+use App\Models\Merchandise;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Migrations\Migration;
+
+class MigrateSwagData extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        FiscalYear::get()->each(static function (FiscalYear $fy): void {
+            if (0 === $fy->packages()->count()) {
+                return;
+            }
+
+            $startingYear = $fy->ending_year - 1;
+            $endingYear = $fy->ending_year;
+            $yearRangeStr = $startingYear.'-'.$endingYear;
+
+            $shirt = Merchandise::firstOrCreate([
+                'name' => 'T-Shirt '.$yearRangeStr,
+                'fiscal_year_id' => $fy->id,
+            ]);
+            $polo = Merchandise::firstOrCreate([
+                'name' => 'Polo '.$yearRangeStr,
+                'fiscal_year_id' => $fy->id,
+            ]);
+
+            $fallPackageName = 'Fall '.$startingYear;
+            $springPackageName = 'Spring '.$endingYear;
+            $studentFullYearName = 'Full Year ('.$startingYear.'-'.$endingYear.')';
+
+            $fallPackage = $fy->packages()->where('name', $fallPackageName)->first();
+            $springPackage = $fy->packages()->where('name', $springPackageName)->first();
+            $studentFullYearPackage = $fy->packages()->where('name', $studentFullYearName)->first();
```

Comment:
```suggestion
            $fallPackage = $fy->packages()->where('name', $fallPackageName)->sole();
            $springPackage = $fy->packages()->where('name', $springPackageName)->sole();
            $studentFullYearPackage = $fy->packages()->where('name', $studentFullYearName)->sole();
```

Reply:
I need this to return `null` if there is no matching package.

Reply:
I think all of those exist, but ok